app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java

```diff
@@ -693,13 +693,6 @@ public static boolean checkRange(final int length, final int... indices) {
         return indices != null && indices.length >= 2 && inRange(0, length, indices) && indices[1] > indices[0];
     }
 
-    public static void blinkView(final View view) {
-        if (view != null) {
-            final float init = view.getAlpha();
-            ObjectAnimator.ofFloat(view, View.ALPHA, init, 0.1f, 1.0f, 0.1f, 1.0f, init).setDuration(1000).start();
-        }
-    }
```

Comment:
Moved this to GsContextUtils. Probably should create GsViewUtils or something.

Reply:
It's OK there 😄